js_modules/dagster-ui/packages/ui-core/src/instigation/InstigationTick.tsx

```diff
@@ -1,56 +1,21 @@
-import {
-  Body,
-  Box,
-  Colors,
-  Group,
-  Icon,
-  NonIdealState,
-  Spinner,
-  Tooltip,
-} from '@dagster-io/ui-components';
+import {Body, Box, Colors, Group, Icon, NonIdealState, Tooltip} from '@dagster-io/ui-components';
+import {useMemo} from 'react';
 
-import {gql, useQuery} from '../apollo-client';
-import {LaunchedRunListQuery, LaunchedRunListQueryVariables} from './types/InstigationTick.types';
+import {gql} from '../apollo-client';
 import {PYTHON_ERROR_FRAGMENT} from '../app/PythonErrorFragment';
-import {RunTable} from '../runs/RunTable';
-import {RUN_TABLE_RUN_FRAGMENT} from '../runs/RunTableRunFragment';
+import {RunsFeedTableWithFilters} from '../runs/RunsFeedTable';
 
 export const RunList = ({runIds}: {runIds: string[]}) => {
-  const queryResult = useQuery<LaunchedRunListQuery, LaunchedRunListQueryVariables>(
-    LAUNCHED_RUN_LIST_QUERY,
-    {
-      variables: {
-        filter: {
-          runIds,
-        },
-      },
-    },
+  const filter = useMemo(
+    () => ({
+      runIds,
+    }),
+    [runIds],
```

Comment:
I noticed this issue in a few other places where we have useQuery with objects as variables. If the component re-renders for any reason we end up re-executing the query. Would be good to clean this up globally. (this was in useRecentAssetEvents but I bet there are more places doing something similar)

Reply:
In general I think we should be memoizing a lot more. using React.memo, etc. In order to make the app snappy and not leave performance foot guns around

Reply:
I'm actually not sure to what extent useQuery does it's own memo magic (it has to do at least some, because `variables: {a, b}` is a new object every time), but agree we should clean these up.